lib/Clam/CfgBuilder.cc

```diff
@@ -2012,6 +2081,7 @@ void CrabIntraBlockBuilder::doGep(GetElementPtrInst &I,
         CRAB_LOG("cfg-gep", crab::outs()
                                 << "-- " << lhs << ":=" << base.getValue()
                                 << "+" << o << "\n");
+        doGepFatAssign(I, bitwidth, lin_exp_t(o));
```

Comment:
As a general comment, I see that you call the fat instrumentation inside `doGep` which is doing the standard translation from LLVM to CrabIR of the GEP instruction. I think it would be good already to call the fat instrumentation outside `doGep` so that it's easier to decouple the standard translation from the fat instrumentation. For that, I think we only need to modify `doGep` to return the crab variable that grabs the offset computation done by doGep.